spec/requests/api/v3/work_package_resource_spec.rb

```diff
@@ -269,7 +269,6 @@
         before(:each) do
           allow(User).to receive(:current).and_return current_user
           work_package
-          ActionMailer::Base.deliveries.clear
```

Comment:
I guess this one was deliberate, to throw away mails from creating the work package.

At least the specs are now failing with one expected and two gotten.